README.md

```diff
@@ -62,78 +61,97 @@ require 'grell'
 crawler = Grell::Crawler.new
 crawler.start_crawling('http://www.google.com') do |current_page|
   if current_page.status == 500 && current_page.retries == 0
-    crawler.restart
+    crawler.manager.restart
     :retry
   end
 end
 ```
 
-### Restarting PhantomJS
-If you are doing a long crawling it is possible that phantomJS starts failing.
-To avoid that, you can restart it by calling "restart" on crawler.
-That will kill phantom and will restart it. Grell will keep the status of
-pages already visited and pages discovered and to be visited. And will keep crawling
-with the new phantomJS process instead of the old one.
+### Pages' id
+
+Each page has an unique id, accessed by the property 'id'. Also each page stores the id of the page from which we found this page, accessed by the property 'parent_id'.
+The page object generated by accessing the first URL passed to the start_crawling(the root) has a 'parent_id' equal to 'nil' and an 'id' equal to 0.
```

Comment:
- space missing after `start_crawling`
- the code-ish mentions should be between back ticks instead of single quotes IMO